lib/ansible/plugins/callback/json.py

```diff
@@ -109,6 +109,7 @@ def _record_task_result(self, on_info, result, **kwargs):
         task_result = result._result.copy()
         task_result.update(on_info)
         task_result['action'] = task.action
+        task_result['tags'] = task.tags
```

Comment:
I don't think tags should go into the result.

Maybe instead:

```
diff --git a/lib/ansible/plugins/callback/json.py b/lib/ansible/plugins/callback/json.py
index c04dd49e23..f6b932e4be 100644
--- a/lib/ansible/plugins/callback/json.py
+++ b/lib/ansible/plugins/callback/json.py
@@ -60,7 +60,8 @@ class CallbackModule(CallbackBase):
         return {
             'task': {
                 'name': task.get_name(),
-                'id': str(task._uuid)
+                'id': str(task._uuid),
+                'tags': task.tags,
             },
             'hosts': {}
```

Reply:
agreed, its not part of the result, it is part of teh task